bundler/index.js

```diff
@@ -4,70 +4,115 @@ const HtmlParser = require('./modules/htmlParser');
+  async create() {
+    try {
+      this._logBundleStarting();
+      this._dist();
+      this._assets();
+      const htmlPath = this.pathConfig.HTML;
+      const htmlData = await this.fileManager.readAggregateFilesData(htmlPath);
+      if (!htmlData) {
+        this._logFileNotCreated(htmlPath);
+        throw `File not created ${htmlPath}`;
+      }
+      const cssBundleName = await this._css(htmlData);
+      const jsBundleName = await this._js(htmlData);
+      await this._html(htmlData, htmlPath, cssBundleName, jsBundleName);
+      this._logBundleFinished();
+    } catch(error) {
+      this._removeDistFolder();
+      this._logBundleFailed();
     }
-    // html
-    await this.processBundleUnit(
-      schema.HTML.input,
-      schema.HTML.output,
-      this.uglifier.uglifyHtml,
-      true
-    );
-    // css
-    await this.processBundleUnit(
-      schema.CSS.input,
-      schema.CSS.output,
-      this.uglifier.uglifyCss
-    );
-    // js
-    await this.processBundleUnit(
-      schema.JS.input,
-      schema.JS.output,
-      this.uglifier.uglifyJs
-    );
-  }
-
-  createSchema(htmlFilePath) {
-    const rawPathes = this.htmlParser.findPathesInHtml(htmlFilePath);
-    
-    const pathes = {};
-    Object.keys(rawPathes).forEach((key) => {
-      pathes[key] = {
-        input: rawPathes[key],
-        output: `dist/bundle.${key.toLowerCase()}`
-      };
-    });
-
-    const split = htmlFilePath.split('/');
-    const htmlFileName = split[split.length - 1];
-    pathes.HTML = {
-      input: [ htmlFilePath ],
-      output: `dist/${htmlFileName}`
-    };
+  }
+
+  _logBundleStarting() {
+    console.log(`⌛ Starting Bundle for ${this.pathConfig.HTML}`)
+  }
+
+  _logFileNotCreated(filePath) {
+    console.log(`🚫 ${filePath}`);
+  }
+
+  _logFileNotFound(filePath) {
+    console.log(`❓ ${filePath}`);
+  }
 
-    return pathes;
+  _logFileCreated(filePath) {
+    console.log(`📁 ${filePath} ✓`)
   }
 
-  async processBundleUnit(filesPathsList, outputPath, uglifyMethod, isHtml = false) {
-    let data = await this.fileManager.readAggregateFilesData(filesPathsList);
-    if (isHtml) {
-      data = this.htmlParser.replaceSources(data);
+  _logBundleFinished() {
+    console.log(`✅ Finished Bundle for ${this.pathConfig.HTML} → ${this.pathConfig.DIST}`)
+  }
+
+  _logBundleFailed() {
+    console.log(`❌ Failed Bundle ${this.pathConfig.HTML}`)
+  }
+
+  _removeDistFolder() {
+    this.fileManager.removeFolder(this.pathConfig.DIST);
+  }
+
+  _write(data, uglifier, filePath) {
+    if (this.fileManager.createWriteFile(uglifier(data), filePath)) this._logFileCreated(filePath);
+    else this._logFileNotCreated(filePath);
+  }
+
+  _dist() {
+    const filePath = this.pathConfig.DIST;
+    this.fileManager.overwriteFolder(filePath);
+    this._logFileCreated(filePath);
+  }
+
+  _assets() {
+    const from = this.pathConfig.ASSETS;
+    const to = this.pathConfig.DIST;
+    const filePath = `${to}/${from}`;
```

Comment:
A tiny one. I think it will improve the readability if we will rename `filePath` to something like `futureFilePath` or `resultFilePath`. 

How do you think? Maybe it's just me. Let me know. :)